irods/test/scripts/pam_irodsA_write_4.3auth.py

```diff
@@ -0,0 +1,5 @@
+import irods.helpers as h
+import irods.client_init as cliinit
+
+cliinit.write_pam_irodsA_file('test123')
+1
```

Comment:
What is the `1` used for? Is it input to something else?

Reply:
Hmm, yes.  I think it was to set a breakpoint.   I think this script will be going away in favor of some real tests, very soon.

Reply:
script removed